JakinChan200/CS180_Roald
# Updated CRUD functionality and data schema to apply to users instead of videos.

## Overview
- Updated the CRUD functionality to apply to users, instead of videos.
- Index/homepage endpoint is now repurposed to handle the CRUD functionality for users.
- Videos are not their own entities anymore, rather they are inherently a property of the users.

## Materials
- #36

## Pictures
Provide a picture or video of your changes if applicable.

## Manual Testing Guide
Send GET, POST, DELETE, and UPDATE HTTP requests (through Postman or cURL) to the /api/ endpoint.
/api/ only accepts GET and POST (only need to read and add users)
/api/:username only accepts DELETE and UPDATE (updates and deletes should not be applied at the collection level)

Pull requests should receive approval before merge.
